php/libraries/NDB_BVL_Instrument.class.inc

```diff
@@ -380,9 +369,7 @@ abstract class NDB_BVL_Instrument extends NDB_Page
                     return true;
                 }
             }
-
-            //no user permissions match required instrument permissions
-            throw new Exception("You do not have access to this page.", 403);
+            return false;
```

Comment:
any value in replacing this with `isAccessibleBy` here or out of scope ?

Reply:
I don't think NDB_BVL_Instrument currently implements the interface, I think renaming the function to implement it would be out of scope here.